kubespawner/proxy.py

```diff
@@ -68,6 +68,13 @@ class KubeIngressProxy(Proxy):
         """
     )
 
+    ingress_host = Unicode(
+        config=True,
+        help="""
+        Hostname for ingress rules
+        """
+    )
+
```

Comment:
I agree with https://github.com/jupyterhub/kubespawner/pull/406#issuecomment-666666196 that we should consider if we have multiple hosts.

This is a bit complicated, because the k8s Ingress resource expose `spec.rules[].host` and `spec.tls.hosts`, so both singular and plural at the same time in a way. I'm not sure what configuration is in scope to define for this though =/

[K8s Ingress resource reference](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.19/#ingress-v1beta1-networking-k8s-io)